light-zk.js/src/transaction/transaction.ts

```diff
@@ -820,12 +820,18 @@ export class Transaction {
       );
 
       let inputsVec = (await coder.encode(accountName, inputs)).subarray(8);
+      // TODO: check whether app account names overlap with system account names and throw an error if so
+      let appAccounts = {};
```

Comment:
shouldnt this be an array?

Reply:
Anchor requires an account object. This way we can just do:
```
tx.accounts({...appAccounts})
```